test/spec/simd.wast

```diff
@@ -766,3 +767,10 @@
 (assert_return (invoke "i32x4.load16x4_u" (i32.const 256)) (v128.const i32x4 0x00009080 0x0000b0a0 0x0000d0c0 0x0000f0e0))
 (assert_return (invoke "i64x2.load32x2_s" (i32.const 256)) (v128.const i64x2 0xffffffffb0a09080 0xfffffffff0e0d0c0))
 (assert_return (invoke "i64x2.load32x2_u" (i32.const 256)) (v128.const i64x2 0x00000000b0a09080 0x00000000f0e0d0c0))
+(assert_return
+  (invoke "v8x16.swizzle"
+    (v128.const i8x16 0xf0 0xf1 0xf2 0xf3 0xf4 0xf5 0xf6 0xf7 0xf8 0xf9 0xfa 0xfb 0xfc 0xfd 0xfe 0xff)
+    (v128.const i8x16 0 4 8 12 16 255 129 128 127 17 15 13 12 8 4 0)
+  )
+  (v128.const i8x16 0xf0 0xf4 0xf8 0xfc 0xf0 0xf0 0xf0 0xf0 0xf0 0xf0 0xff 0xfd 0xfc 0xf8 0xf4 0xf0)
```

Comment:
Hmm. According spec:
```
if s[i] < S.lanes:
  result[i] = a[s[i]]
else:
  result[i] = 0
```
So I guess result should be `0xf0 0xf4 0xf8 0xfc` **`0x00 0x00 0x00 0x00 0x00 0x00`** `0xff 0xfd 0xfc 0xf8 0xf4 0xf0`

Reply:
Aha, you're right. I mentally translated "For indices outside of the range the resulting lane is 0" to " For indices outside of the range the resulting lane is *lane* 0." Thanks for the catch!

Reply:
Yeah, always welcome)